### ISSUE TITLE
docker/dockerfile_generator.sh: No such file or directory

### ISSUE THREAD

**Issue by M-Davies** [AdoptOpenJDK/openjdk-build#1909]
**Platform:**
Mac 10.15.5
**Architecture:**
x64

[docker/dockerfile_generator.sh isn't found](https://github.com/AdoptOpenJDK/openjdk-build/blob/master/docker-build.sh#L86)
```
➜  openjdk-build git:(master) ./makejdk-any-platform.sh --docker jdk8u
Starting ./makejdk-any-platform.sh to configure, build (Adopt)OpenJDK binary
Parsing opt: --docker
Possible opt arg: jdk8u
Working dir is ./build/
[debug] COPY_MACOSX_FREE_FONT_LIB_FOR_JDK_FLAG=true
[debug] COPY_MACOSX_FREE_FONT_LIB_FOR_JRE_FLAG=true
JDK Image folder name: j2sdk-bundle/jdk*.jdk
JRE Image folder name: j2re-bundle/jre*.jre
Searching for JDK_BOOT_DIR
Guessing JDK_BOOT_DIR: /Library/Java/JavaVirtualMachines/adoptopenjdk-14-openj9.jdk/Contents/Home
If this is incorrect explicitly configure JDK_BOOT_DIR
Boot dir set to /Library/Java/JavaVirtualMachines/adoptopenjdk-14-openj9.jdk/Contents/Home
/Users/morgan.davies@ibm.com/Documents/Repos/openjdk-build/docker-build.sh: line 86: docker/dockerfile_generator.sh: No such file or directory
```

**Comment by M-Davies** [comment]
@Willsparker By going back in the commit history and running the command again, it looks like it was caused by https://github.com/AdoptOpenJDK/openjdk-build/pull/1864

Does [docker/dockerfile_generators.sh](https://github.com/AdoptOpenJDK/openjdk-build/pull/1864/files#diff-1e5c4882847e3eefa76de60a300139b8) need to be a full path?

**Comment by Willsparker** [comment]
Sorry- an oversight in my PR you linked above 😓  Should be fixed with #1910
